server/models.py

```diff
@@ -44,8 +49,53 @@ class Organization(BaseModel):
     name = Column(String(200), nullable=False, unique=True)
 
 
-def get_or_404(model: Type[Base], primary_key: str):
-    instance = model.query.get(primary_key)
-    if instance:
-        return instance
-    raise NotFound(f"{model.__class__.__name__} {primary_key} not found")
+class AdminUser(BaseModel):
+    id = Column(String(200), primary_key=True)
+    email = Column(String(200), unique=True, nullable=False)
+    external_id = Column(String(200), unique=True)
+
+    organization_id = Column(
+        String(200), ForeignKey("organization.id", ondelete="cascade"), nullable=False
+    )
+
+
+class Election(BaseModel):
+    id = Column(String(200), primary_key=True)
+    name = Column(String(200), nullable=False)
+
+    organization_id = Column(
+        String(200), ForeignKey("organization.id", ondelete="cascade"), nullable=False
+    )
+
+    definition_file_id = Column(String(200), ForeignKey("file.id", ondelete="cascade"))
+    definition = Column(JSON)
+
+    voter_file_id = Column(String(200), ForeignKey("file.id", ondelete="cascade"))
```

Comment:
might be overkill right now to track the files since we're not going to upload a huge voter file and the election definition is a few K at the most. We could start by just storing the definition raw and not tracking all the individual voter files.

Reply:
Ok, that works for me